src/test/java/com/googleinterns/zoomtube/servlets/AuthenticationServletTest.java

```diff
@@ -81,11 +82,7 @@ public void doGet_loggedIn_expectFalse() throws Exception {
 
   @Test
   public void doGet_returnsUserEmail() throws Exception {
-    final String EMAIL = "test@example.com";
-    authService.setEnvIsLoggedIn(true);
-    authService.setEnvAuthDomain("example.com");
-    authService.setEnvEmail(EMAIL);
```

Comment:
This is an important part of the test and therefore should be done here. If someone was reading these tests they would need to know that they need to look at setup to find where the email was set.

Reply:
Duplicated to relevant test